jumpscale/sals/reservation_chatflow/deployer.py

```diff
@@ -1545,5 +1552,27 @@ def wait_pool_payment(self, bot, pool_id, exp=5, qr_code=None, trigger_cus=0, tr
 
         return False
 
+    def get_qr_code_payment_info(self, pool):
+        escrow_info = pool.escrow_information
+        resv_id = pool.reservation_id
+        escrow_address = escrow_info.address
+        escrow_asset = escrow_info.asset
+        total_amount = escrow_info.amount
+        total_amount_dec = Decimal(total_amount) / Decimal(1e7)
+        thecurrency = escrow_asset.split(":")[0]
+        total_amount = "{0:f}".format(total_amount_dec)
+        qr_code = f"{thecurrency}:{escrow_address}?amount={total_amount}&message=p-{resv_id}&sender=me"
+        msg_text = f"""
+        
+        <h4> Wallet Address: </h4>  {escrow_address} \n
+        <h4> Currency: </h4>  {thecurrency} \n
+        <h4> Memo Text (Reservation Id): </h4>  p-{resv_id} \n
+        <h4> Total Amount: </h4> {total_amount} {thecurrency} \n
+
+        <h5>Inserting the memo-text is an important way to identify a transaction recipient beyond a wallet address. Failure to do so will result in a failed payment. Please also keep in mind that an additional Transaction fee of 0.1 FreeTFT will automatically occurs per transaction.</h5>
```

Comment:
seems you have 0.1 FreeTFT hardcoded in the fee part. should be 0.1 {thecurrency}